project/Game.py

```diff
@@ -71,9 +73,57 @@ def test_completed_game(self):
         self.game.end()
         self.assertTrue(self.game.ended, "Game Has Ended")
 
+class TestDeleteLandmarks(unittest.TestCase):
+    def setUp(self):
+        self.game = Game()
+
+    def test_delete_landmark(self):
+        landmark1 = "ABC"
+        self.game.add_landmark(landmark1)
```

Comment:
Instead of calling the add_landmark method you should force it to be in there (we're not testing add_landmark here)